ezomero/_gets.py

```diff
@@ -1152,12 +1152,24 @@ def get_map_annotation(conn: BlitzGateway, map_ann_id: int,
     --------
     >>> ma_dict = get_map_annotation(conn, 62)
     >>> print(ma_dict)
-    {'testkey': 'testvalue', 'testkey2': 'testvalue2'}
+    {'testkey': 'testvalue', 'testkey2': ['testvalue2'. 'testvalue3']}
     """
     if type(map_ann_id) is not int:
         raise TypeError('Map annotation ID must be an integer')
+    
+    map_annotation_dict = {}
+    
+    map_annotation = conn.getObject('MapAnnotation', map_ann_id).getValue()
+
+    for item in map_annotation:
+        if item[0] in map_annotation_dict:
+            if not isinstance(map_annotation_dict[item[0]], list):
+                map_annotation_dict[item[0]] = [map_annotation_dict[item[0]]]
```

Comment:
hmmm, yes.
that is a smoother way to write this.
I was not aware of this feature.
I would even say it makes more sense to get a list for everything, so you always get the same data-type.
Something like:
```
map_annotation_dict = defaultdict(list)
for item in map_annotation:
	map_annotation_dict[item[0]].append(item[1])
```
But I would leave the decision about this up to you, as it is your library.

Reply:
I think using `defaultdict`s and always getting a list might make more sense (and results in a more consistent API). It is a breaking change for the API, so it might take a bit longer to make it to release, but I think it makes future maintenance easier, so we should probably do that.